lib/src/over_react_test/jacket.dart

```diff
@@ -115,7 +114,7 @@ class TestJacket<T extends react.Component> {
   /// >   ```
   /// >   queryByTestId(jacket.mountNode, yourTestId)
   /// >   ```
-  ReactComponent getInstance() {
+  ReactComponent? getInstance() {
```

Comment:
I think this return should be non-nullable, since the 
```dart
if (!_isCompositeComponent && /*[1]*/!_isDomComponent) {
````
check below throws if `_renderedInstance` is null.